cms/djangoapps/contentstore/views/component.py

```diff
@@ -210,6 +211,7 @@ def container_handler(request, usage_key_string):
                 'xblock_info': xblock_info,
                 'draft_preview_link': preview_lms_link,
                 'published_preview_link': lms_link,
+                'keywords_supported': [{'name':keyword, 'desc': value.desc} for keyword, value in keyword_substitution.KEYWORD_FUNCTION_MAP.iteritems()],
```

Comment:
I know this convention is used throughout the platform, but I don't care for single-line comprehensions.

They always seem more difficult to read and hence more prone to bugs.

Could we split this up like:
```python
'keywords_supported': [
    {
        'name': keyword,
        'desc': value.desc,
    }
    for keyword, value in keyword_substitution.KEYWORD_FUNCTION_MAP.iteritems()
]
```